concordium-sdk/src/main/java/com/concordium/sdk/ClientV2MapperExtensions.java

```diff
@@ -460,4 +475,244 @@ static CCDAmount to(Amount amount) {
+
+        switch (payload.getPayloadCase()) {
+            case DEPLOY_MODULE: {
+                final WasmModule deployModulePayload = to(payload.getDeployModule());
+                return DeployModuleTransaction
+                        .builderBlockItem()
+                        .header(to(transaction.getHeader(), deployModulePayload.getBytes().length))
+                        .signature(to(transaction.getSignature()))
+                        .payload(deployModulePayload)
+                        .build();
+            }
+            case INIT_CONTRACT: {
+                final InitContractPayload initContractPayload = to(payload.getInitContract());
+                return InitContractTransaction.builderBlockItem()
+                        .header(to(transaction.getHeader(), initContractPayload.getBytes().length))
+                        .signature(to(transaction.getSignature()))
+                        .payload(initContractPayload)
+                        .build();
+            }
+            case UPDATE_CONTRACT:
+                final UpdateContractPayload updateContractPayload = to(payload.getUpdateContract());
+                return UpdateContractTransaction
+                        .builderBlockItem()
+                        .header(to(transaction.getHeader(), updateContractPayload.getBytes().length))
+                        .signature(to(transaction.getSignature()))
+                        .payload(updateContractPayload)
+                        .build();
+            case REGISTER_DATA:
+                final Data registerDataPayload = to(payload.getRegisterData());
+                return RegisterDataTransaction
+                        .builderBlockItem()
+                        .header(to(transaction.getHeader(), registerDataPayload.getBytes().length))
+                        .signature(to(transaction.getSignature()))
+                        .payload(registerDataPayload)
+                        .build();
+            case TRANSFER:
+                final TransferPayload transferPayload = to(payload.getTransfer());
+                return TransferTransaction
+                        .builderBlockItem()
+                        .header(to(transaction.getHeader(), transferPayload.getBytes().length))
+                        .signature(to(transaction.getSignature()))
+                        .payload(transferPayload)
+                        .build();
+            case TRANSFER_WITH_MEMO:
+                final TransferWithMemoPayload transferWithMemoPayload = to(payload.getTransferWithMemo());
+                return TransferWithMemoTransaction
+                        .builderBlockItem()
+                        .header(to(transaction.getHeader(), transferWithMemoPayload.getBytes().length))
+                        .signature(to(transaction.getSignature()))
+                        .payload(transferWithMemoPayload)
+                        .build();
+            case RAW_PAYLOAD:
```

Comment:
All the items not covered in the above enum are being sent back as Raw Payload. 
Should this be deserialized using the byte array ?

Reply:
I don't think we should spend time on that now. So what you have is fine I think. 
Maybe rename the "fake" type from `UNKNOWN_ACCOUNT` to something more describing i.e. `RAW_PAYLOAD`.

Reply:
done